### ISSUE TITLE
Create a complete and complex sample project using  django-extra-views

### ISSUE THREAD

**Issue by luzfcb** [AndrewIngram/django-extra-views#52]
Please, create a complete and complex sample project using  django-extra-views.

django-extra-views supports nested inlines, similar to the django-nested-inline[1] makes?

[1] https://github.com/silverfix/django-nested-inlines

**Comment by sdolemelipone** [comment]
As part of the examples/test cases we should switch from `__unicode__` to `__str__` and other python 3 conventions.

**Comment by yerycs** [comment]
So, django-extra-view supports nested inline?

**Comment by CleitonDeLima** [comment]
> So, django-extra-view supports nested inline?

Yes, I use it in my project. But the validations are done on a Formset base class made by me.

**Comment by yerycs** [comment]
Could you share code snippet @CleitonDeLima ?

**Comment by CleitonDeLima** [comment]
> Could you share code snippet @CleitonDeLima ?

https://gist.github.com/CleitonDeLima/b1530b3d45bf1576cedb4056e1aa1df1

`django-extra-views` is not changed.
